packages/dear-imgui/cpp/app/src/imgui_renderer.cpp

```
@@ -382,8 +385,41 @@ void ImGuiRenderer::SetCurrentContext() {
+    if (file_size == -1) {
+        printf("Unable to determine file size of image\n");
+    }
+
+    fseek(f, 0, SEEK_SET);
+
+    void* file_data = IM_ALLOC(file_size);
+
+    fread(file_data, 1, file_size, f);
+
+    m_reactImgui->m_imageToTextureMap[job.widgetId] = LoadTexture(file_data, file_size);
+};
```

cppcheck

[error] Resource leak

(resourceLeak)